bots/zephyr_mirror_backend.py

```diff
@@ -596,13 +622,14 @@ def zcrypt_encrypt_content(zephyr_class, instance, content):
     return encrypted
 
 def forward_to_zephyr(message):
+    # type: (Dict[str, Any]) -> Any
```

Comment:
Then its is giving this error `Jains-Mac-Pro:zulip jain-PC$ tools/run-mypy
bots/zephyr_mirror_backend.py: note: In function "forward_to_zephyr":
bots/zephyr_mirror_backend.py:686: error: No return value expected
bots/zephyr_mirror_backend.py:709: error: No return value expected
bots/zephyr_mirror_backend.py:725: error: No return value expected
bots/zephyr_mirror_backend.py:738: error: No return value expected``